src/features/content-explorer/content-explorer-modal-container/__tests__/ContentExplorerModalContainer.test.js

```diff
@@ -6,8 +7,8 @@ import ContentExplorerModalContainer from '../ContentExplorerModalContainer';
 describe('features/content-explorer/content-explorer-modal-container/ContentExplorerModalContainer', () => {
     const sandbox = sinon.sandbox.create();
     const initialSelectedItems = { '123': { id: '123', name: 'folder123' } };
-    const renderComponent = props =>
-        shallow(
+    const renderComponent = (renderer = shallow, props) =>
```

Comment:
```suggestion
    const renderComponent = (props, renderer = shallow) =>
```
usually the default props are always to the right so that you are not required to fill the parameter when you call the function, like you have below.  So you can also remove the `shallow` as a parameter below with this change